app/controllers/App.scala

```diff
@@ -48,10 +50,24 @@ class App(val wsClient: WSClient, val atomWorkshopDB: AtomWorkshopDBAPI) extends
         atomType <- validateAtomType(atomType)
         ds <- AtomDataStores.getDataStore(atomType, Preview)
         atom <- atomWorkshopDB.createAtom(ds, atomType, req.user)
+        _ <- sendKinesisEvent(atom, previewAtomPublisher, EventType.Update)
       } yield atom
     }
   }
 
+  def publishAtom(atomType: String, id: String) = AuthAction { req =>
```

Comment:
I think this is a bit more complicated that this unfortunately (chatted with Phil in person)